src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-background-color;
+  box-sizing: border-box;
```

Comment:
try to add paddings to the card instead of adding margin-left/right for elements

Reply:
not fixed

Reply:
But i have padding-inline: 16px; or you mean something else?